CodeEdit/Features/StatusBar/Views/StatusBarDrawer/StatusBarDrawer.swift

```diff
@@ -27,7 +33,24 @@ struct StatusBarDrawer: View {
     var body: some View {
         VStack(spacing: 0) {
             switch model.selectedTab {
-            case 0: TerminalEmulatorView(url: model.workspaceURL)
+            case 0:
+                TerminalEmulatorView(url: model.workspaceURL)
+                    .background {
+                        if colorScheme == .dark {
```

Comment:
I feel like this could be done cleaner but I'm still pretty new to Swift and SwiftUI critiques are much appreciated! :)

Reply:
Yeah I agree, but I think the AppPreferences datamodel should be reworked to make code cleaner here. i don't think you can do much here

Reply:
@matthijseikelenboom can you elaborate what can be improved?

Reply:
@lukepistrol Yeah sure. The thing that I think is most confusing, is the multiple models thing. Let me give an example from themes. There are models in the root folder and in the section folder.

There is:
CodeEdit/CodeEdit/Features/AppPreferences/Model/Theme/ThemePreferences.swift
CodeEdit/CodeEdit/Features/AppPreferences/Sections/ThemePreferences/Model/Theme.swift
CodeEdit/CodeEdit/Features/AppPreferences/Sections/ThemePreferences/Model/ThemeModel.swift

Also there is a lot of `prefs.preferences.theme.selectedTheme` where `prefs.prefereneces` is weird to me, but that might also be that `prefs` is just not a good naming of that variable. 

`if prefs.preferences.theme.selectedTheme == prefs.preferences.theme.selectedLightTheme {` This piece of code gets called a lot as a whole I believe, so perhaps we should have these in calculated variables.

(I see now that you wrote it haha. With the code it's self is nothing wrong and well documented)

Reply:
@matthijseikelenboom Yeah I see what you mean 😉 To be honest I'm not quite happy with the naming either haha

Reply:
Should this hold back this PR though or is it within scope?

Reply:
I agree

Reply:
> Should this hold back this PR though or is it within scope?

@austincondiff Not really in scope, could be done with a separate PR and issue

> I agree

@lukepistrol With what?

Reply:
@matthijseikelenboom That it's out of scope for this PR